### PR TITLE
add inline and proxy scanner resources

### PR DESCRIPTION
**Pull request by credibleforce**
***Issue***: https://lacework.atlassian.net/jira/software/projects/ALLY/issues/ALLY-1037

***Description:***
Added inline scanner and proxy scanner resources.

***Additional Info:***
Releated to go-sdk pull request https://github.com/lacework/go-sdk/pull/1018

### CODE REVIEW

File: integration/resource_lacework_integration_inline_scanner_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			"name": tokenName,
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new Github Container Registry
+	create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+	createData := GetContainerRegisteryInlineScanner(create)
+	assert.Equal(t, tokenName, createData.Data.Name)
+	// assert.Equal(t, []map[string]string{{"foo": "bar"}}, createData.Data.Data.IdentifierTag)
```
Comment (dmurray-lacework) [inline integration/resource_lacework_integration_inline_scanner_test.go:33]: Would be great to have this assertion included.

File: integration/resource_lacework_integration_inline_scanner_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/resource_lacework_integration_inlineScanner",
+		EnvVars:      tokenEnvVar,
+		Vars: map[string]interface{}{
+			"name": tokenName,
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new Github Container Registry
```
Comment (dmurray-lacework) [inline integration/resource_lacework_integration_inline_scanner_test.go:29]: Would be nice to have some TFstate assertions too. 

There should be some examples in other tests. 
`actualID := terraform.Output(t, terraformOptions, "name")`
requires adding outputs to your example/main.tf file

Comment (credibleforce) [reply integration/resource_lacework_integration_inline_scanner_test.go:29]: added terraform assertions to test the output. good catch!

File: integration/resource_lacework_integration_inline_scanner_test.go (hunk 1 above)
Comment (credibleforce) [reply integration/resource_lacework_integration_inline_scanner_test.go]: added back these assertion tests

**Comment by credibleforce** [comment]
@dmurray-lacework hoping you can take a look at this change, updated to reference latest go-sdk

**Comment by afiune** [comment]
make it so

**Comment by credibleforce** [comment]
@afiune any chance you could give this another go?

**Review by afiune — COMMENTED** [review]
I'm sorry @credibleforce but I can't review this PR in its current state, I have pulled down your change and rebased it here https://github.com/lacework/terraform-provider-lacework/pull/434

Please, make sure that everything is as it should be. Meaning that, check I didn't miss anything.

**Comment by afiune** [comment]
closing in favor of https://github.com/lacework/terraform-provider-lacework/pull/434

**Pull request closed without merging**